### PR TITLE
Run eastwood in CI

### LINKED ISSUE #12: Add eastwood to CI
https://github.com/jonase/eastwood

eastwood sometimes produces false positives, so perhaps it should not be gating. Keep in mind that eastwood does _active_ analysis, i.e. it reads your code, so that means it is not okay for code to have import-time side effects.

### PR DESCRIPTION
**Pull request by reaperhulk**
Fixes #12, depends on #13.

Suggestions welcome for the env var name. `EASTWOOD=false` is kind of annoying for "every test except eastwood"

### CODE REVIEW

**Comment by lvh** [comment]
Let's see what it does first. It could be that a `GATING_TESTS=true/false` makes sense.

File: project.clj [hunk 1]
```diff
@@ -6,7 +6,10 @@
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [org.onyxplatform/onyx "0.8.2"]]
   :plugins [[lein-cljfmt "0.3.0"]
-            [lein-cloverage "1.0.7-SNAPSHOT"]]
+            [lein-cloverage "1.0.7-SNAPSHOT"]
+            [lein-kibit "0.1.2"]
+            [lein-kibit "0.1.2"]
```
Comment (lvh) [inline project.clj]: This dependency is in there twice for reasons I'm going to assume is a side effect of this depending on an earlier PR?

Comment (reaperhulk) [inline project.clj]: No that'd be my incompetence.

**Comment by lvh** [comment]
I don't understand why this passed, since kibit is supposed to be gating, and [it does find a number of errors](https://travis-ci.org/RackSec/desdemona/jobs/106342072#L324-L345).

Another issue is that the if block appears to produce all output as 1 command, which makes it trickier to see what command produced what output. Would a shell script with the appropriate echoing flags help?

**Comment by lvh** [comment]
LGTM

**Pull request merged**